core/src/test/java/kafka/server/share/SharePartitionManagerTest.java

```diff
@@ -1246,26 +1250,40 @@ public void testCloseSharePartitionManager() throws Exception {
     @Test
     public void testReleaseAcquiredRecordsSuccess() {
         String groupId = "grp";
-        String memberId = Uuid.randomUuid().toString();
+        Uuid memberId = Uuid.randomUuid();
```

Comment:
Any particular reason toString method is removed here and then used at multiple places within the test wherever memberId is being used?

Reply:
Yeah, a conversion from string to Uuid was required later so don't see any point in converting to string first then back to Uuid from string.